docs/CONTRIBUTING.md

```diff
@@ -7,7 +7,7 @@ TODO: CZI to update to match specific git working model.
 This project uses [Git-flow](https://nvie.com/posts/a-successful-git-branching-model/) to manage the development workflow. Here's a breakdown of what this means for the design system:
 
 - The `main` always represents the latest production-ready version of the design system. Every commit to `main` indicates a new release.
-- `next` represents the unpublished next Major release of the design system. All feature work that belongs to the new architecture  TODO: revisit the existance of `next` once the new architecture is in place and launched
+- `next` represents the unpublished next Major release of the design system. **use this branch as the base of your changes**.
```

Comment:
2 things:
- you can use develop off `main` directly if it's a bugfix that you want to put out quickly without messing with publishing everything currently on `next`
- this was already in there but I'm wondering if "Major" release makes sense here because the next release off `next` could be minor or patch (or maybe I'm misunderstanding what this line is trying to say 🤔 )

Reply:
@dierat yah you are right, GitFlow specifies how hot fixes can be based off `main` to expedite things (which can get merged back into `next`). I think "Major" in this sentence means closer to "substantial"? Even then, we could have one commit on `next` that fixes a typo so ...

Honestly i think inverting these bullets would be clearer, e.g., "for urgent hot fixes, etc., base work off of `main` branch" and so on. Right now, they only allude to which branch to use.

Reply:
I would also be fine with tweaking the copy to include the hotfix exemption. Maybe something like

>- The `main` always represents the latest production-ready version of the design system. Every commit to `main` indicates a new release. The only commits made to this branch should be regular releases and hotfixes.
>- `next` represents the unpublished next Major release of the design system. **Use this branch as the base of your changes** for regular changes (hotfixes can be branched off `main`.